diagnostic.md

```diff
@@ -91,9 +106,14 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
+
+
+
+My answer:
+
+color: yellow;
+font-family: sans-serif
```

Comment:
Because the element has the inline style of font-family: cursive, this element will have font-family: cursive, because inline styles have higher specificity than styles in a css document.